tests/upgrade/test_existence.py

```diff
@@ -0,0 +1,139 @@
+"""Upgrade TestSuite for validating Satellite entities existence and
```

Comment:
Maybe having this on a separated directory from tests would be better, something like upgrade_tests at the root project's directory.